### PR TITLE
Add cookieName parameter

### PR DESCRIPTION
**Pull request by allexks**
Implementing #51

### CODE REVIEW

**Comment by allexks** [comment]
Sorry for the failing pipeline, I am new to the JS / React world. I will create tests for the new feature whenever as soon as I can.

**Comment by xavierbriole** [comment]
Thank you for the PR 🙏 
You're the first contributor of this repository! 🎉 

Dont worry, I will review these changes and help you pass the pipeline! 🚀

File: src/Validator/index.js [hunk 1]
```diff
@@ -122,3 +122,11 @@ export const validateDarkTheme = (parameter?: any): boolean => {
 
   return window.matchMedia('(prefers-color-scheme: dark)').matches
 }
+
+export const validateCookieName = (parameter?: any): string => {
```
Comment (xavierbriole) [inline src/Validator/index.js:126]: Can you place this function just before the `validateDarkTheme()` function please ?

File: src/Components/CookieNotice.js [hunk 2]
```diff
@@ -73,6 +74,7 @@ type Props = {|
   readMoreButtonLink?: string,
   openInNewTab?: boolean,
   cookieTextLabel?: string,
+  cookieName?: string,
```
Comment (xavierbriole) [inline src/Components/CookieNotice.js:77]: Can you place this line between `cookieExpiration` prop and `darkTheme` prop please ?

File: src/Components/CookieNotice.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ import {
   validateMaxWidth,
   validateCookieExpiration,
   validateDarkTheme,
+  validateCookieName,
```
Comment (xavierbriole) [inline src/Components/CookieNotice.js:21]: And this line just below `validateCookieExpiration` import ! (I like ordering lol)

**Review by xavierbriole — CHANGES_REQUESTED** [review]
That's good! For the pipeline, just run `yarn run format`if you don't have prettier in your IDE.

Just waiting for the tests and let's go for the next version !

**Comment by allexks** [comment]
I wrote tests for the validator but am not sure how to write tests for `CookieNotice.js` to cover the lines suggested by CodeCov, could you help with this?

**Comment by xavierbriole** [comment]
Hmmm, yes I will check asap.

**Comment by xavierbriole** [comment]
@allexks hello, I wrote CookieNotice tests to be 100% code coverage on the project 🎉 

Can you please rebase your branch, adapt your tests if needed and check if the pipeline pass ?

File: src/Components/CookieNotice.test.js [hunk 4]
```diff
@@ -141,12 +146,16 @@ describe('CookieNotice', () => {
 
   describe('should set cookie', () => {
```
Comment (xavierbriole) [inline src/Components/CookieNotice.test.js:147]: Can you replace this test suite into a single test `it('should set cookie', () => {})` that mock : 
```js
mock(validateCookieExpiration).mockImplementation(() => 'mockedCookieExpiration')
mock(validateCookieName).mockImplementation(() => 'mockedCookieName')
```

**Review by xavierbriole — CHANGES_REQUESTED** [review]
Sorry for the response/review latence, I don't have a lot of time ahah.
I did some last reviews. Thanks !!

**Review by xavierbriole — APPROVED** [review]

**Pull request merged**